packages/forklift-console-plugin/src/modules/Providers/views/details/components/DetailsSection/components/NameAndWebUiLinkDetailsItem.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { EditProviderUIModal, useModal } from 'src/modules/Providers/modals';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { ExternalLink } from '@kubev2v/common';
+import { V1beta1Provider } from '@kubev2v/types';
+import { DescriptionListDescription } from '@patternfly/react-core';
+
+import { DetailsItem } from '../../../../../utils';
+
+import { ProviderDetailsItemProps } from './ProviderDetailsItem';
+
+/**
+ * @typedef {Object} NameDetailsItemProps - extends ProviderDetailsItemProps
+
+ * @property {string} [webUILinkText - A text to be displayed as a content.
+ * @property {function} [calcWebUILink] - A method for auto calculating the provider's UI path.
+ **/
+
+export interface NameAndWebUiLinkDetailsItemProps extends ProviderDetailsItemProps {
+  webUILinkText?: string;
+  calcWebUILink?: (provider: V1beta1Provider) => string;
+}
+
+const getProviderUIAnnotation = (provider: V1beta1Provider): string =>
+  provider?.metadata?.annotations?.['forklift.konveyor.io/providerUI'];
+
+/**
+ * Component for displaying the provider name details item.
+ * It includes a title and two content items:
+ * 1. The provider name (non-editable)
+ * 2. The provider web UI link (editable, can be auto calculated or taken from YAML)
+ *
+ * @component
+ * @param {DetailsItemProps} props - The props of the details item.
+ */
+export const NameAndWebUiLinkDetailsItem: React.FC<NameAndWebUiLinkDetailsItemProps> = ({
+  resource: provider,
+  moreInfoLink,
+  helpContent,
+  canPatch,
+  webUILinkText,
+  calcWebUILink,
+}) => {
+  const { t } = useForkliftTranslation();
+  const { showModal } = useModal();
+
+  const defaultMoreInfoLink =
+    'https://kubernetes.io/docs/concepts/overview/working-with-objects/names';
+  const defaultHelpContent = t(
+    "This field includes the name and an optional provider's web UI link. Name is primarily intended for creation idempotence and configuration definition. Cannot be updated. Provider's web UI link is editable and its value is either taken from YAML or auto calculated.",
```

Comment:
```suggestion
    "A client-provided string that refers to an object in a resource URL, such as /api/v1/pods/some-name.

Only one object of a given kind can have a given name at a time. However, if you delete the object, you can make a new object with the same name.

Use the link below to access the web-based user interface for virtual machine management. You can store the link to the management system in the Provider metadata annotations value under the 'forklift.konveyor.io/providerUI' annotation.
```

Reply:
> "A client-provided string that refers to an object in a resource URL" 

?? 
not clear.

Reply:
> not clear.

I'ts coppied from https://kubernetes.io/docs/concepts/overview/working-with-objects/names, yes, they are not that clear, but it's a text the k8s team docs think is good, I'm ok with adding some information it you think it's needed to make it more clear, this text has lots of space.